src/utils/moduleNamespace.js

```diff
@@ -32,7 +32,10 @@ export function moduleNamespace(root, module) {
       return;
     }
 
-    moduleNamespace = parts[1].trim();
+    moduleNamespace = parts[1]
+      .trim()
+      .split(" ")[0]
```

Comment:
be careful. `as*with` could be written without spaces (as `*` is not a valid identifier character, spaces are optional)

Reply:
Sure. And since we're already talking about it, there are other cases such as:

```scss
// With line breaks
@use 'sass:map'
  as
  *
  with ();
```
or:
```scss
// Using 'with' as namespace
@use 'sass:map'
  as
  with
  with ();
```

That seems to escalate a bit,
but I will have a look at it.